### DISCUSSION TITLE
why does "lists" need to take up my screen space?

### DISCUSSION THREAD

**Discussion by chakravala** [community/community#37671] [Other Feature Feedback, Questions, & Ideas]
here we go, Microsoft is again trying to ruin github and turn it into a facebook or linkedin

I don't want  half the screen space taken up by an empty feature box I don't plan to use

who are these microsoft developers coming up with these stupid ideas? seriously, half the screen taken up by an empty "list" box?

**Comment by jfullstackdev** [comment]
do you mean like this view ? 

![image](https://user-images.githubusercontent.com/47092464/198883341-b1f6be7b-a2cf-4dd7-93aa-472ec4803643.png)

**Reply by atlslscsrv-app** [reply]
I have no idea 💡 how you can still be polite and not even mention to this individual how maybe if they weren't so disrespectful maybe Microsoft Devs would stop messing up there space? 
And give them an option for a solution.
I need to learn how you do it.
